src/Catrobat/Commands/ImportLegacyCommand.php

```diff
@@ -183,13 +183,12 @@ protected function execute(InputInterface $input, OutputInterface $output): void
         $num = count($data);
         if ($num > 2)
```

Comment:
why have you changed this legacy command?

Reply:
Because I needed language version inside FeaturedProgram isn't that obvious?????

Reply:
I'm just asking because I never used this Command before + there is a ticket to remove it.
That's why I'm bewildered about changes in this file.